### PR TITLE
External Compaction Blog post

### PR DESCRIPTION
**Pull request by keith-turner**
Adds a blog post about the new external compaction feature and a test run using
the new feature.

Co-authored-by: Dave Marion <dlmarion@apache.org>

### CODE REVIEW

File: _posts/blog/2021-06-21-external-compactions.md [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+Server. The external compaction feature is an extension of the major compaction
+service in the Tablet Server and is configured as part of the systems
+compaction service configuration. Thus, it is an optional feature. The goal of
+the external compaction feature is to overcome some of the drawbacks of the
+Major compactions that happen inside the Tablet Server. Specifically, external
+compactions:
+
+ * Allow major compactions to continue when the originating TabletServer dies
+ * Allow major compactions to occur while a Tablet migrates to a new Tablet Server
+ * Reduce the load on the TabletServer, giving it more cycles to insert mutations and respond to scans (assuming it’s running on different hosts).  Map reduce jobs and compactions can lower the effectiveness of processor caches for scans, so moving them off can be beneficial.
```
Comment (dlmarion) [inline _posts/blog/2021-06-21-external-compactions.md:39]: Could add page cache in addition to processor cache

**Comment by dlmarion** [comment]
FYI - for easier reading with inline graphs, use https://github.com/keith-turner/accumulo-website/blob/extcomp/_posts/blog/2021-06-21-external-compactions.md

File: _posts/blog/2021-06-21-external-compactions.md (hunk 1 above)
Comment (keith-turner) [reply _posts/blog/2021-06-21-external-compactions.md:39]: Could say something like `processor caches (L1,L2,L3, and TLB)`. Maybe this covers data and page caches.. or maybe `processor data and page caches`

Comment (dlmarion) [reply _posts/blog/2021-06-21-external-compactions.md]: Updated in b256256

**Review by dlmarion — APPROVED** [review]

**Pull request merged**